encode/Makefile.am

```diff
@@ -36,6 +36,14 @@ h264encode_LDADD	= \
 	$(top_builddir)/common/libva-display.la \
 	-lpthread -lm
 
+hevcencode_SOURCES	= hevcencode.c
+hevcencode_CFLAGS	= -I$(top_srcdir)/common -g
+hevcencode_LDADD	= \
+       	$(LIBVA_LIBS) \
+	$(top_builddir)/common/libva-display.la \
```

Comment:
better to keep alignment with previous

Reply:
Hevc has the same alignment with h264.
Could you tell me which line is different?